utils/k8s/openebscrd.go

```diff
@@ -40,3 +40,26 @@ func (k8s K8S) DeleteStoragePoolClaim(spcName string) error {
 	spcClient := k8s.OpenebsClientSet.Openebs().StoragePoolClaims()
 	return spcClient.Delete(spcName, &meta_v1.DeleteOptions{})
 }
+
+// GetCStorPool returns the CStorPool object for given cStorPoolName.
+// :return: *openebs_v1.CStorPool: Pointer to CStorPool objects.
+func (k8s K8S) GetCStorPool(cStorPoolName string) (*openebs_v1.CStorPool, error) {
+	cStorPoolClient := k8s.OpenebsClientSet.Openebs().CStorPools()
+	return cStorPoolClient.Get(cStorPoolName, meta_v1.GetOptions{})
+}
+
+// ListCStorPool returns all CStorPool objects.
+func (k8s K8S) ListCStorPool() ([]openebs_v1.CStorPool, error) {
```

Comment:
We want to do list like following:
https://github.com/openebs/CITF/pull/50/files#diff-0838b5a1a898271846f11064eaefaab7R45